bitgo-java-api/src/main/java/com/bitso/bitgo/v2/BitGoClientImpl.java

```diff
@@ -104,14 +104,28 @@ public BitGoClientImpl(String token, String baseUrl, boolean unsafe) {
     }
 
     @Override
-    public Optional<SendCoinsResponse> sendMany(String coin, String walletId, String walletPass,
-                                                Map<String, BigDecimal> recipients,
-                                                String sequenceId, String message,
-                                                BigDecimal fee, BigDecimal feeTxConfirmTarget,
-                                                int minConfirms, boolean enforceMinConfirmsForChange)
-            throws IOException {
-        String url = baseUrl + SEND_MANY_URL.replace("$COIN", coin).replace("$WALLET", walletId);
+    public Optional<SendCoinsResponse> sendMany(Map<String, Object> parameters) throws IOException {
+        // Get all needed parameters
+        String coin = parameters.get("coin").toString();
+        String walletId = parameters.get("walletId").toString();
+        String walletPass = parameters.get("walletPass").toString();
```

Comment:
See, the doc specifies these are required parameters, but I don't see any validation here to enforce that (I know, it's being delegated to bitgo, but why do that if it can be enforced beforehand?) – if a parameter is missing from the map there will be a NPE which sheds no light on what's missing, and if the value is present but empty, we're sending an invalid request to bitgo.